### DISCUSSION TITLE
How do remove domain from blacklist?

### DISCUSSION THREAD

**Discussion by ozon2021** [MetaMask/eth-phishing-detect#59568] [Q&A] [answered]
Hi!
https://github.com/MetaMask/eth-phishing-detect/issues/59238
https://github.com/MetaMask/eth-phishing-detect/issues/59023
https://github.com/MetaMask/eth-phishing-detect/issues/11325
how to remove from the blacklist?

https://urlscan.io/search/#rig-coin.com
https://github.com/MetaMask/eth-phishing-detect/pull/10848

**Comment by SirDivider** [comment, marked as answer]
go to support
